### ISSUE TITLE
remove commented source code

### ISSUE THREAD

**Issue by gocarlos** [cose-wg/COSE-C#87]
e.g `EC_FromKey` in `mbedtls.c`

It would probably make sense to remove code, which is commented out since 4 years.

@jimsch is this code needed?

**Comment by jimsch** [comment]
No, that was cut and paste from the openssl code when I was putting in the ECDH code for MBEDTLS
